### ISSUE TITLE
UnusedVariable: some false positives

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#438]
String interpolation. Code to reproduce:
```php
$bar = helper('foo');
       
$this->something("Baz $bar(quoz)");
```

Calling a method. Code to reproduce:
```php
if ($foo) {
    $method = 'foo';
} else {
    $method = 'bar';
}

\Log::$method('Lorem ipsum...');
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/a1c24840b34b2fcdd7cc1e4d00d3b857c6715a9c
